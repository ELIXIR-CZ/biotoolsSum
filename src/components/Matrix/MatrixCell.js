import * as React from 'react'
import { Image } from 'react-bootstrap'

export const MatrixCell = ({ image, text }) => {
  return (
    <div className='square dark-grey-background'>
      <div className='square-content'>
        <div className='table'>
          <div className='table-cell text-big'>
            {image && <Image src={image} rounded className='responsive-img' />}
            {text && <span>{text}</span>}
          </div>
        </div>
      </div>
    </div>
  )
}
