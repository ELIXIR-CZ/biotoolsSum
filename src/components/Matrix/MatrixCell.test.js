import React from 'react'
import renderer from 'react-test-renderer'
import MatrixCell from './MatrixCell'

it('Renders MatrixCell correctly', () => {
  const component = renderer.create(
    <MatrixCell text='This is some text' />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
