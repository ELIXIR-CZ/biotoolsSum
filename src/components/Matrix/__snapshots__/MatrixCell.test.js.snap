exports[`test Renders MatrixCell correctly 1`] = `
<div
  className="square dark-grey-background">
  <div
    className="square-content">
    <div
      className="table">
      <div
        className="table-cell text-big">
        <span>
          This is some text
        </span>
      </div>
    </div>
  </div>
</div>
`;
